encoder/libav_encoder.cpp

```diff
@@ -180,9 +180,8 @@ void LibAvEncoder::initVideoCodec(VideoOptions const *options, StreamInfo const
 	const char *format = nullptr;
 	if (options->libav_format.empty())
 	{
-		if (options->output.empty() ||
-			options->output.find(tcp.c_str(), 0, tcp.length()) != std::string::npos ||
-			options->output.find(udp.c_str(), 0, udp.length()) != std::string::npos)
+		if (output_file_.empty() || output_file_.find(tcp.c_str(), 0, tcp.length()) != std::string::npos ||
+			output_file_.find(udp.c_str(), 0, udp.length()) != std::string::npos)
```

Comment:
Did the style checker request this change? I think I preferred the original!

Reply:
Yes.  I prefer it as the original as well, I'll revert it.